www/templates/www/index.html

```diff
@@ -25,8 +25,8 @@ <h1 class="display-3">
 </div>
 
 <div class="container">
-  <div class="alert alert-info" role="alert">
-    {% blocktrans %}<strong>Note</strong> This membership system is still under heavy development. If you find bugs or inconsistensies please report them. Thanks!{% endblocktrans %}
-  </div>
+  {% blocktrans %}<p>Hello! This is member registration system for Tampere Hacklab ry. If you have not been member before you can fill out our member application by clicking "Apply membership in hacklab" button. If you already are member of the community, you can log in by clicking top right corner "Log in" button. In case you are rejoining to our crew, please contact hallitus by email and wait for the instructions.</p>
+  <p>With this system you can manage your member information and invoices. When you log in first time, please check that your information is correct.</p>
+  <p>This membership system is still under heavy development. If you find bugs or inconsistencies please report them for the team mulysa. Thank you in advance!</p>{% endblocktrans %}
```

Comment:
- get site name (tampere hacklab) from settings
 - show membership application text only to users that are not logged in
 - show "In case you are rejoining to our crew, please contact hallitus by email and wait for the instructions." only if user is logged in and has only suspended member services.
 - member registration system  -> membership management service

I suggest splitting to multiple translation strings.

Reply:
Should have probably read through the text before commiting :)